pygmt/base_plotting.py

```diff
@@ -153,14 +153,15 @@ def coast(self, **kwargs):
         D="position",
         F="box",
         G="truncate",
+        I="shading",
         W="scale",
         V="verbose",
         X="xshift",
         Y="yshift",
         p="perspective",
         t="transparency",
     )
-    @kwargs_to_strings(R="sequence", G="sequence", p="sequence")
+    @kwargs_to_strings(R="sequence", G="sequence", p="sequence", I="sequence")
```

Comment:
We usually put upper-case options before lower-case options.
```suggestion
    @kwargs_to_strings(R="sequence", G="sequence", I="sequence", p="sequence")
```